api/rooms/views/main.py

```diff
@@ -0,0 +1,22 @@
+import requests, json
+
+from django.views     import View
+from django.http      import JsonResponse
+
+from my_settings import API_KEY
+
+class SearchRoomView(View):
+    def get(self, request):
+        try:
+            search_address = request.GET.get('search_address', None)
+            search_address = search_address.strip()
```

Comment:
`strip`을 사용하시는 이유가 있나요?
```suggestion
            search_address = search_address.strip()
```

Reply:
검색시 공백제거를 위해서 썼는데 혹시 다른 방법이 있을까요?